packages/teams-js/view-published-packages-urls.ps1

```diff
@@ -1,4 +1,4 @@
-$version = Get-ChildItem -Path $(System.DefaultWorkingDirectory)/_OfficeDev.microsoft-teams-library-js/CDNFeed -Directory -Name
+$version = Get-ChildItem -Path ../CDNFeed -Directory -Name
```

Comment:
This relies on CDNFeed to be present at parent layer. Would it be possible to provide the parent directory as param?

Reply:
I think Anupam makes a good suggestion. The pipeline originally used an inline script (not a file) and we were able to evaluate the variable that way. I'm curious if you can just pass the entire directory parent of CDNFeed to the script, as Anupam suggests, like:

"$(System.DefaultWorkingDirectory)/_OfficeDev.microsoft-teams-library-js"

(Maybe I'm missing something, but I don't follow why the script being published to the directory you mention would make passing a variable to the script not feasible. I'm happy to learn more!)

Reply:
Sorry it was me that made the mistake. I saw it as part of the script itself rather than as a param and I think that will work. 

The only thing that comes to mind of why it might not work is if the Powershell this script is being run in doesn't have the AAD specific environmental variables. Which seems to be the case with running the current instance of the compliant Powershell. This can be solved by rmaking a Get-Module request.

Reply:
Just ran this and sending the $(System.DefaultWorkingDirectory) as an argument/parameter into the script worked :)